Directory.Build.props

```diff
@@ -1,3 +1,6 @@
 <Project>
+  <PropertyGroup>
+    <LombiqHelpfulLibrariesPath>$(MSBuildThisFileDirectory)Libraries\Lombiq.HelpfulLibraries\Lombiq.HelpfulLibraries.OrchardCore\Lombiq.HelpfulLibraries.OrchardCore.csproj</LombiqHelpfulLibrariesPath>
+  </PropertyGroup>
```

Comment:
Wrong repo, this should be [here](https://github.com/Lombiq/DotNest-Tenants-Core/blob/issue/NEST-319/Directory.Build.props).

Reply:
I think if I put it inside the main project's `Directory.Build.props` this one overrides it, deleting the value. I tried this and gives me nothing: 
![image](https://user-images.githubusercontent.com/80963259/182461386-c12f3213-ab66-49b7-a75d-c3cea9eeb5f3.png)
![image](https://user-images.githubusercontent.com/80963259/182461401-b0c7a2a3-e9a8-4a36-83a5-2febbb19ef8e.png)

When I put it in this `Directory.Build.props` I can see the path in the console.

Reply:
I found [this documentation](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2022#search-scope). What a horrible system. Then put the property back into the OrchardCore.Commerce.csproj and let's get this over with.

I learn something bad about MSBuild every day smh...